packages/dev/core/src/Engines/abstractEngine.ts

```diff
@@ -2655,4 +2657,64 @@ export abstract class AbstractEngine {
      * By default, this will create a Database object if the workload has been embedded.
      */
     public static OfflineProviderFactory: (urlToScene: string, callbackManifestChecked: (checked: boolean) => any, disableManifestCheck: boolean) => IOfflineProvider;
+
+    /**
+     * Will flag all materials in all scenes in all engines as dirty to trigger new shader compilation
+     * @param flag defines which part of the materials must be marked as dirty
+     * @param predicate defines a predicate used to filter which materials should be affected
+     */
+    public static MarkAllMaterialsAsDirty(flag: number, predicate?: (mat: Material) => boolean): void {
+        for (let engineIndex = 0; engineIndex < EngineStore.Instances.length; engineIndex++) {
+            const engine = EngineStore.Instances[engineIndex];
+
+            for (let sceneIndex = 0; sceneIndex < engine.scenes.length; sceneIndex++) {
+                engine.scenes[sceneIndex].markAllMaterialsAsDirty(flag, predicate);
+            }
+        }
+    }
+
+    // Updatable statics so stick with vars here
+
+    /**
+     * Gets or sets the epsilon value used by collision engine
+     */
+    public static CollisionsEpsilon = 0.001;
+
+    /**
+     * Find the next highest power of two.
+     * @param x Number to start search from.
+     * @returns Next highest power of two.
+     */
+    public static CeilingPOT: (x: number) => number = CeilingPOT;

```

Comment:
Maybe all those should be called from their root location instead ?

Reply:
Do you mean move the method's implementation to a child class?

By the way. these methods seem to be legacy and not used anywhere

Reply:
Ok let;s remove then from there then. I checked the PGs and no one are using them

Reply:
Removed. Should I update the breaking changes documents?

Reply:
nope all good. We consider that 0 hit on more 300000 pgs means that the functions were private :)